packages/flutter/test/animation/animation_controller_test.dart

```diff
@@ -187,6 +187,8 @@ void main() {
     expect(controller.value, moreOrLessEquals(0.0));
     controller.stop();
 
+    controller.dispose();
+
```

Comment:
I noticed that in another test, the controller was being reassigned without a `.dispose()` call, so I added this as well.